cmd/operator/deploy/crds/monitoring.googleapis.com_operatorconfigs.yaml

```
@@ -93,6 +93,16 @@ spec:
             type: object
             description: Features holds configuration for optional managed-collection features.
             properties:
+              config:
+                type: object
+                description: Configuration of the Prometheus configuration.
```

Bit weird wording, maybe 
```suggestion
                description: Settings for the collector configuration propagation.
```